build-config/src/main/resources/weld-checkstyle/checkstyle.xml

```diff
@@ -50,9 +50,9 @@
             <property name="ignoreNumbers" value="-1, 0, 1, 2, 3"/>
         </module>
         <module name="MissingSwitchDefault" />
-        <module name="RedundantThrows">
+        <!--<module name="RedundantThrows">
```

Comment:
I have described the reason in commit message : https://github.com/weld/build/pull/11/commits/48fac59ea2264a745a5905c46f1d94b6ca8798bb
RedundantModifier could be uncommented but it should be configured differently or code has to be cleaned since it behavior changed a lot between checkstyle 6.1.1.
BTW it's not very convenient to have checkstyle config and checkstyle lib version definition in different projects ;)